.github/workflows/ci.yml

```diff
@@ -105,7 +105,7 @@ jobs:
 
     - name: Install Python packaging tools
       run: |
-        python -m pip install --upgrade pip setuptools wheel
+        python -m pip install --upgrade pip setuptools wheel numpy
```

Comment:
NumPy should be installed later as a declared dependency - you shouldn't have needed to change this here (same comment below).

Reply:
Without NumPy, the `python setup.py sdist --formats=gztar,zip` command fails because I use NumPy in the setup.py file to get the location of the C API header files (`np.get_include()`). This location is `numpy/core/include/numpy`. Maybe we could hardcode this, but I think NumPy would still need to be installed in order to compile the NumPy extension module(s).